pkg/web/static/styles/traveladdress.css

```diff
@@ -0,0 +1,41 @@
+#traddr-encoder-form {
+    max-width: 600px;
+}
+
+#traddr-encoder-form textarea,
+#traddr-encoder-form button,
+#traddr-encoder-form div.output-container {
+  margin: 24px auto;
+  display: block;
+}
+
+#traddr-encoder-form div.output-container,
+#traddr-encoder-form textarea {
+    width: 100%;
+    min-width: 300px;
+    min-height: 150px;
+    border: 2px solid #333;
+    border-radius: 6px;
+    padding: 12px;
+}
+
+#output-label {
+  text-align: center;
+  margin: -26px auto 12px auto;
+  background-color: #FFF;
+  width: 84px;
+  font-weight: bold;
+}
+
+.error-text {
+  margin: -16px 0 4px 0;
+}
+
+textarea:focus {
+  outline: none !important;
+  border-color: #55ACD8 !important;
```

Comment:
```suggestion
  border-color: #3B82F6 !important;
```

Reply:
This color is used in other inputs throughout the UI, so this suggestion is for consistency. Also, it has a better contrast against the white background.